pkgs/development/libraries/qt-6/modules/qtwebengine.nix

```diff
@@ -143,6 +152,9 @@ qtModule {
       --replace "AppleClang" "Clang"
     substituteInPlace cmake/Functions.cmake \
       --replace "/usr/bin/xcrun" "${xcbuild}/bin/xcrun"
+    # Disable metal shader compilation, Xcode only
```

Comment:
Nitpick. Can we make this substitution and the `AppleClang` one running on all platforms? Than we won't break macos if the package is updated.

Reply:
Done